### DISCUSSION TITLE
Create Bookmarks is Blocked

### DISCUSSION THREAD

**Discussion by sayborduu** [3kh0/ext-remover#557] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I can’t create bookmarks. They’re blocked by the admin.

Any way to get around this?

**Comment by ehunt34** [comment, marked as answer]
Paste the javascript directly into the URL bar.

**Reply by sayborduu** [reply]
I tried some days ago, blocked also

**Reply by ehunt34** [reply]
Likely they put "javascript://*" into your URLBlocklist Chrome policy. Can't fix that buddy.

**Reply by sayborduu** [reply]
They did that haha, just the “javascript:”

**Reply by sayborduu** [reply]
They blocked it because of me lmao
